src/gateway/WebSocketClient.ts

```diff
@@ -0,0 +1,128 @@
+import WebSocket from 'ws';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { EVENTS, Payload } from './constants';
+import { GATEWAY_OPCODES, GATEWAY_CLOSE_EVENT_CODES } from '../util/Constants';
+
+export default class WebSocketClient {
+  public IDENTIFY_TIMEOUT = 4000;
```

Comment:
any reason this is public?
I would honestly have this as a constant outside of the class entirely (like right above the class)
At the very least, private and readonly.

Reply:
done